pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -357,7 +361,7 @@ func (c *Compactor) compactTenant(ctx context.Context, logger log.Logger, sc sto
 	// Tokenizer is not thread-safe so we need one per goroutine.
 	NGramLength := c.limits.BloomNGramLength(tenant)
 	NGramSkip := c.limits.BloomNGramSkip(tenant)
-	bt, _ := v1.NewBloomTokenizer(c.reg, NGramLength, NGramSkip)
+	bt, _ := v1.NewBloomTokenizer(NGramLength, NGramSkip, c.btMetrics)
```

Comment:
We should check the returned error.

Reply:
I removed the error return value instead. There's wasn't anything to return an error in `NewBloomTokenizer `, creates a map, creates . that's why we never checked it anywhere, seems like an afterthought.